src/lib/Components/HoursCollapsible.tsx

```diff
@@ -77,13 +77,15 @@ export class HoursCollapsible extends React.Component<Props, State> {
   render() {
     const { isExpanded } = this.state
     return (
-      <Box mt={2}>
+      <Box>
         <TouchableWithoutFeedback onPress={this.handleToggleIsExpanded}>
-          <Flex justifyContent="space-between" alignItems="center" flexDirection="row">
+          <Flex alignItems="center" flexDirection="row">
             <Sans size="3t" weight="medium">
-              Hours
+              Opening hours
             </Sans>
-            <Chevron source={require("../../../images/chevron.png")} isExpanded={isExpanded} />
+            <Box ml={1}>
+              <Chevron source={require("../../../images/chevron.png")} isExpanded={isExpanded} />
```

Comment:
I think we should consider using the svg version if only for consistency sake

Reply:
cool I think this will need to be refactored in several places, I can do this in a separate PR?

Reply:
yup sounds good